package.json

```diff
@@ -21,21 +21,21 @@
     "David Chan <david@troi.org>"
   ],
   "devDependencies": {
-    "qunit": "0.9.1",
+    "eslint-config-wikimedia": "0.1.0",
     "grunt": "1.0.1",
     "grunt-cli": "1.2.0",
-    "grunt-contrib-jshint": "1.0.0",
     "grunt-contrib-connect": "1.0.1",
     "grunt-contrib-qunit": "1.2.0",
     "grunt-contrib-watch": "1.0.0",
-    "grunt-jscs": "2.8.0"
+    "grunt-eslint": "18.0.0",
```

Comment:
You still run tests on .10 so you can't use `grunt-eslint` since it requires node > 4